### PR TITLE
Updated discovery/readme

### PR DESCRIPTION
**Pull request by madpah**
Update to consistently refer to the final portion of the TEI as `unique-identifier`.

### CODE REVIEW

**Comment by madpah** [comment]
Agree @oej - let me take another pass.

**Comment by madpah** [comment]
Few tweaks @oej - I think the clarity was already there tbh - but have added some more and a couple of TEI examples.

File: discovery/readme.md [hunk 1]
```diff
@@ -60,44 +59,64 @@ urn:tei:<type>:<domain>:<unique-identifier>
...
 * SHA512
 
 ```text
-urn:tei:hash:<domain or host>:<hashtype>:<hash>
+urn:tei:hash:<domain-name>:<hashtype>:<hash>
 ````
 
+Example:
+```text
+urn:tei:org.cyclonedx:SAH256:fd44efd601f651c8865acf0dfeacb0df19a2b50ec69ead0262096fd2f67197b9
```
Comment (oej) [inline discovery/readme.md:107]: s/SAH256/SHA256/

File: discovery/readme.md [hunk 2]
```diff
@@ -34,20 +34,19 @@ to global uniqueness without new registries.
...
-- The **type** which defines the syntax of the unique identifier part
-- The **domain name** part does not have to exist as a web server (HTTPS).
-- The uniqueness of the name is the domain name part that has to be registred at creation of the TEI.
-- The **unique identifier** has to be unique within the domain. Recommendation is to use UUID,
-  but it can be an existing article code too.
+- The **`type`** which defines the syntax of the unique identifier part
+- The **`domain-name`** part does not have to exist as a web server (HTTPS).
+  - The uniqueness of the name is the domain name part that has to be registred at creation of the TEI.
+- The **`unique-identifier`** has to be unique within the `domain-name`. 
+  - Recommendation is to use UUID, but it can be an existing article code too.
```
[the remark this answers is not in the document]
Comment (madpah) [reply discovery/readme.md:41]: That statement was previously there - I did not add it @oej! Will clarify.

File: discovery/readme.md (hunk 1 above)
Comment (madpah) [reply discovery/readme.md]: Correction pushed.

File: discovery/readme.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (madpah) [reply discovery/readme.md]: See update @oej

[the remark this answers is not in the document]
Comment (oej) [reply discovery/readme.md]: Ha ha, was it my old mistake that I did not agree with?

**Pull request merged**